__tests__/app.test.js

```diff
@@ -147,6 +143,39 @@ describe("GET/api/articles/:article_id", () => {
         expect(res.body.message).toBe("Bad Request");
       });
   });
+  test("response with the status of 200 and array of all the available comments for the given article", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+        expect(body.comments).toBeSortedBy("created_at");
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            body: expect.any(String),
+          });
```

Comment:
This is a really good start but dont forget we also want to ensure the array is not empty so before the forEach you will need to assert that the array has atleast one element within it. We also should be expecting the article_id for all of these to be 3 as that those are the comments we are expecting back.